lua/astrocommunity/docker/lazydocker/init.lua

```diff
@@ -0,0 +1,23 @@
+local prefix = "<Leader>t"
+return {
+  {
+    "mgierada/lazydocker.nvim",
+    dependencies = {
+      "akinsho/toggleterm.nvim",
+      {
+        "AstroNvim/astrocore",
+        opts = {
+          mappings = {
+            n = {
+              [prefix .. "d"] = {
+                function() require("lazydocker").open() end,
+                desc = "ToggleTerm LazyDocker",
+              },
+            },
+          },
+        },
+      },
+    },
+    config = function() require("lazydocker").setup {} end,
```

Comment:
```suggestion
    config = function() require("lazydocker").setup {} end,
```
This is not needed. Use opts instead.